src/main/scripts/src/app/shared/perspectives/icon-picker/palette/icon-picker-palette.component.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+
+@Component({
+  selector: 'palette',
+})
+export class PaletteComponent implements OnInit, OnDestroy {
```

Comment:
Is there any intention to use this class alone? If not, it should be made abstract and it does not need `@Component` decorator.